src/main/resources/application.yaml

```diff
@@ -95,7 +95,6 @@ idam:
   client:
     id: ${IDAM_CLIENT_ID:sptribs-case-api}
     redirect_uri: ${IDAM_CLIENT_REDIRECT_URI:http://localhost:3001/receiver}
-    secret: ${IDAM_CLIENT_SECRET:thUphEveC2Ekuqedaneh4jEcRuba4t2t}
```

Comment:
Only the secret itself needs to be removed, not the config entry

Reply:
Please erase commit history after merge so this secret is no longer visible in git history

Reply:
`secret: ${IDAM_CLIENT_SECRET:DUMMY_SECRET} `